ingester/ingester.go

```diff
@@ -131,9 +133,19 @@ func New(cfg Config, chunkStore cortex.Store) (*Ingester, error) {
 		),
 		chunkUtilization: prometheus.NewHistogram(prometheus.HistogramOpts{
 			Name:    "cortex_ingester_chunk_utilization",
-			Help:    "Distribution of stored chunk utilization.",
+			Help:    "Distribution of stored chunk utilization (when stored).",
 			Buckets: []float64{0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9},
```

Comment:
Didn't know about that (I don't think I added this / not sure).  But nice, will use in the future.